.rubocop.yml

```diff
@@ -0,0 +1,40 @@
+AllCops:
+  Exclude:
+    - "bin/*"
+    - "db/**/*"
+    - "config/**/*"
+    - "Guardfile"
+    - "Rakefile"
+  DisplayCopNames: true
+
+Metrics/LineLength:
+  Max: 120
+Metrics/MethodLength:
+  Include: ["app/controllers/*"]
+  Max: 20
+Metrics/AbcSize:
+  Include: ["app/controllers/*"]
+  Max: 30
+Metrics/ClassLength:
+  Max: 150
+Metrics/BlockLength:
+  ExcludedMethods: ["describe"]
+Metrics/ModuleLength:
+  Max: 150
+Metrics/CyclomaticComplexity:
+  Max: 100
+Metrics/PerceivedComplexity:
+  Max: 100
+
```

Comment:
**[REQUIRED]**
We need to remove these lines and use only the lines provided in the [original](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml) .rubocop.yml file provided by Microverse.

Reply:
When I use the normal one, I keep getting errors that the complexity of some of the methods are too high. I don't know how else to write them